protos/configs.proto

```diff
@@ -189,6 +221,17 @@ message ActionConfig {
     // the  action depends on data from a source which has not been declared as
     // a dependency.
     bool hermetic = 20;
+
+    // Supported file formats for BigQuery tables.
+    enum FileFormat {
```

Comment:
`FileFormat` logically belongs to `IcebergTableConfig`. I propose to move the enum definition there.